src/components/Card.js

```diff
@@ -45,15 +83,39 @@ export default class Card {
     // get card view
     const cardImage = this._cardElement.querySelector(".card__image");
     const cardCaption = this._cardElement.querySelector(".card__caption");
+    const likeButton = this._cardElement.querySelector(".card__like-button");
 
     cardImage.src = this._link;
     cardImage.alt = this._alt;
-    cardCaption.textContent = this._place;
+    cardCaption.textContent = this._name;
 
-    // set event listener
+    if (this._isLiked) {
+      likeButton.classList.add("card__like-button_active");
+    }
+    this._cardElement.dataset.id = this._id;
     this._setEventListeners();
-
-    // return card
     return this._cardElement;
   }
 }
+
+export async function renderInitialCards(
+  cardSelector,
+  handleImageClick,
+  currentUserId
+) {
+  try {
+    const result = await api.getInitialCards();
+    result.forEach((data) => {
```

Comment:
You need to use method `renderItems` of class `Section` for rendering the array of cards